Plugins/RSBot.Map/Views/Main.cs

```diff
@@ -356,7 +357,7 @@ private void PopulateMapAndGrid(Graphics graphics)
 
         private void DrawCollisions(Graphics gfx)
         {
-            if (CollisionManager.HasActiveMeshes)
+            if (CollisionManager.HasActiveMeshes && GlobalConfig.Get("RSBot.EnableCollisionDetection", true))
```

Comment:
use `CollisionManager.Enabled` instead of GlobalConfig

Reply:
Done